docs/How-do-I-run-my-own.md

```diff
@@ -21,7 +21,7 @@ Get yourself a Linux server (they come as cheap as $2/month), and then make sure
 
 ##### Verify it runs
 
-Test DNSChain by simply running `dnschain` from the command line (developers [see here](#Working)). Have a look at the configuration section below, and when you're ready, run it in the background as a daemon. As a convenience, DNSChain [comes with a `systemd` unit file](<../scripts/dnschain.service>) that you can use to run it.
+Test DNSChain by simply running `dnschain` from the command line (developers [see here](#Developers.md#Working)). Have a look at the configuration section below, and when you're ready, run it in the background as a daemon. As a convenience, DNSChain [comes with a `systemd` unit file](<../scripts/dnschain.service>) that you can use to run it.
```

Comment:
I think the `#` in front of `Developers.md` needs to be removed.

You can amend your commit to fix and force push the replacement for this PR.